vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java

```diff
@@ -66,11 +66,25 @@ public MenuBarTestPage() {
                 e -> menuBar.setOpenOnHover(!menuBar.isOpenOnHover()));
         openOnHoverButton.setId("toggle-open-on-hover");
 
-        NativeButton setWidthButton = new NativeButton("set width 150px",
-                e -> setWidth("150px"));
+        NativeButton setWidthButton = new NativeButton("set width 150px", e -> {
+            setWidth("140px");
+            menuBar.getElement().callFunction("notifyResize");
+        });
         setWidthButton.setId("set-width");
 
+        NativeButton resetWidthButton = new NativeButton("reset width", e -> {
+            setWidth("initial");
```

Comment:
Default value for the `width` property is `auto`.

Reply:
done